core/primitives-core/src/version.rs

```diff
@@ -19,7 +19,7 @@ pub enum ProtocolFeature {
     /// Always use gas price computed in the previous block.
     FixApplyChunks,
     _DeprecateLowerStorageCost,
-    DeleteActionRestriction,
+    _DeprecatedDeleteActionRestriction,
```

Comment:
Any reason to keep these around vs. just deleting?

Nit, the one above is `_Deprecate` and this is `_Deprecate`**d**.

Reply:
I was thinking of keeping these around just to have an easy way to see historic features. It would be easy for someone to blame and navigate to the time the feature was removed, otherwise it's hard to know how really to search for this.

Also, I'll have _Deprecated to _Deprecate in future PRs

Reply:
I misunderstood the comment.. Changed to consistent `_Deprecated` with the d